react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -348,3 +352,109 @@ function getLogProperty(
     if (log_value) return { name: prop_name, value: log_value };
     else return null;
 }
+
+function CatmullRom(
+    p0: number,
+    p1: number,
+    p2: number,
+    p3: number,
+    t: number
+): number {
+    const t2 = t * t;
+    const t3 = t * t * t;
+    return  0.5 *(2*p1 + (-p0 + p2) * t + (2*p0 - 5*p1 + 4*p2 - p3) * t2 + (-p0 + 3*p1- 3*p2 + p3) * t3); // eslint-disable-line
```

Comment:
Why disable eslint? If it's because the line is too long, better split the formula on multiple lines.